mypy.ini

```diff
@@ -1,2 +1,5 @@
+[mypy]
+ignore_missing_imports = True
```

Comment:
This is pretty broad, is it needed?

Reply:
Yes, complains about subprocess32 in py3 version